newm-server/src/test/kotlin/io/newm/server/features/song/SongRoutesTests.kt

```diff
@@ -789,6 +789,9 @@ class SongRoutesTests : BaseApplicationTests() {
         val response = client.get("v1/songs/$songId/stream") {
             bearerAuth(testUserToken)
             contentType(ContentType.Application.Json)
+            url {
```

Comment:
Have to explicitly set the test protocol to HTTPS in order to satisfy the Secure cookie check.